system/jlib/jsocket.cpp

```diff
@@ -416,8 +416,10 @@ class CSocket: public ISocket, public CInterface
     bool        set_nagle(bool on);
     void        set_linger(int lingersecs); 
     void        set_keep_alive(bool set);
+    void        logConnectionInfo(unsigned timeoutms, unsigned conn_mstime);
     virtual void set_inherit(bool inherit=false);
     virtual bool check_connection();
+    virtual bool isSecure() const;
```

Comment:
trivial: I add override to every overriden method I can these days, it's already saved mishaps a few times and in cases where overriden was missing and the base was changed it's caused problems that weren't noticed.